frontend/src/components/elements/NavbarElement/MobileNavbarComponent.tsx

```diff
@@ -1,67 +1,166 @@
-import React from 'react';
-import Router from 'next/router';
+import React, {useState} from 'react';
+import Router, {NextRouter} from 'next/router';
 import styles from './navbar.mobile.module.css';
-import PersonRoundedIcon from '@mui/icons-material/PersonRounded';
-import TuneRoundedIcon from '@mui/icons-material/TuneRounded';
-import LogoutRoundedIcon from '@mui/icons-material/LogoutRounded';
 import Button from '@mui/material/Button';
-import {navbarProps, buttonProps} from './NavbarComponentInterface';
+import {navbarProps} from './NavbarComponentInterface';
+import {Backdrop, SvgIcon, Tooltip} from '@mui/material';
+import DateRangeIcon from '@mui/icons-material/DateRange';
+import SensorsRoundedIcon from '@mui/icons-material/SensorsRounded';
+import PeopleAltRoundedIcon from '@mui/icons-material/PeopleAltRounded';
+import RouteIcon from '@mui/icons-material/Route';
+import ApartmentRoundedIcon from '@mui/icons-material/ApartmentRounded';
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded';
+import MenuRoundedIcon from '@mui/icons-material/MenuRounded';
+
+const topButtons = [
```

Comment:
The mockups you made don't have top buttons, instead only showing an image of the Dr Trottoir logo. I don't think this list of buttons is needed, as the pages we want to show are already in botButtons.